### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by kamilatokarska**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,21 +65,27 @@ You should now be able to use **Tag Master** with LiveChat.
...
-In order to pull data from our server, Rest API requires you to include an **access_token** in all the requests. You can get it using [LiveChat Boilerplate](https://docs.livechatinc.com/boilerplate/) and [JavaScript Widget API](https://docs.livechatinc.com/agent-app-widgets/#javascript-api).
+In order to pull data from our server, you need to include an **access_token** in all the requests. You can get it using one of the [agent authorization flows](https://developers.livechat.com/docs/authorization/agent-authorization).
 
-You should also get familiar with [Authorization](https://docs.livechatinc.com/authorization/).
+# More sample apps
+
+Experiment more with our different sample apps:
+ - [Progress](https://github.com/livechat/progress-app) - widget for helping you monitor statistics of your team, such as chat ratings, response times, and chatting times.
+ - [Supervisor](https://github.com/livechat/supervisor-app) - widget for helping you monitor weekly progress of your agents and also their availability.
+ - [Sample auth app with the redirect flow](https://github.com/livechat/sample-app-redirect-auth)
```
Comment (arturfracala) [inline README.md]:
```suggestion
 - [Sample auth app with the authorization redirect flow](https://github.com/livechat/sample-app-redirect-auth)
```

or 

```suggestion
 - [Sample auth app with the OAuth redirect flow](https://github.com/livechat/sample-app-redirect-auth)
```

 `Sample auth app with the redirect flow` doesn't tell much

File: README.md [hunk 2]
```diff
@@ -65,21 +65,27 @@ You should now be able to use **Tag Master** with LiveChat.
...
+ - [Sample auth app with the popup flow](https://github.com/livechat/sample-app-popup-auth)
```
Comment (arturfracala) [inline README.md]: Same story here

Comment (kamilatokarska) [inline README.md]:
```suggestion
 - [Sample app with the authorization popup flow](https://github.com/livechat/sample-app-popup-auth)
```

Comment (kamilatokarska) [inline README.md]:
```suggestion
 - [Sample auth app with the authorization popup flow](https://github.com/livechat/sample-app-popup-auth)
```

Comment (kamilatokarska) [inline README.md]:
```suggestion
 - [Sample Popup App with the popup authorization flow](https://github.com/livechat/sample-app-popup-auth)
```

File: README.md (hunk 1 above)
Comment (kamilatokarska) [inline README.md]:
```suggestion
 - [Sample Redirect App with the redirect authorization flow](https://github.com/livechat/sample-app-redirect-auth)
```

**Pull request merged**